src/index.tsx

```diff
@@ -12,12 +12,23 @@ type Modules = ReturnType<qrcodegen.QrCode['getModules']>;
 type Excavation = {x: number; y: number; w: number; h: number};
 type ErrorCorrectionLevel = 'L' | 'M' | 'Q' | 'H';
 
-const ERROR_LEVEL_MAP: {[index: string]: qrcodegen.QrCode.Ecc} = {
+type ERROR_LEVEL_MAPPED_TYPE = {
+  [index in ErrorCorrectionLevel]: qrcodegen.QrCode.Ecc;
+};
```

Comment:
Improved the type of `ERROR_LEVEL_MAP` to `Mapped Types` and added `as const` to make it safer to use as a constant.